core/provisioning-java/src/main/java/org/apache/syncope/core/provisioning/java/data/UserDataBinderImpl.java

```diff
@@ -668,4 +679,13 @@ public UserTO getUserTO(final String key) {
         return getUserTO(userDAO.authFind(key), true);
     }
 
+    private boolean isMustChangePwdMappingItemConfigured(Mapping mapping) {
+        return IterableUtils.matchesAny(mapping.getItems(), new Predicate<MappingItem>() {
+            @Override
+            public boolean evaluate(MappingItem mappingItem) {
```

Comment:
`final` is missing here, build will fail

Reply:
added it.